### DISCUSSION TITLE
Question about updating to ➔ astroid-package-v3.0.7.Framework-Template - Normal tha both Zero/one Templates stay 3.0.5?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#513] [Q&A]
Question about updating to ➔ _astroid-package-v3.0.7.Framework-Template_ - Normal that both Zero/one Templates stay 3.0.5?

![Screen Shot 2023-12-21 at 9 09 44 PM](https://github.com/templaza/astroid-framework/assets/1907276/4be2fd03-6174-4f1d-b97b-8a14ad4ecf49)

**Comment by nidzo80** [comment]
No
![2023-12-22_135143](https://github.com/templaza/astroid-framework/assets/21330927/0de1fcbe-4d66-4702-a73d-7416e28cfa9e)

**Comment by salahjaafar** [comment]
Yes, I've detected that. It's necessary to install the templates separately in order to migrate the version.

**Comment by Chacapamac** [comment]
In my case, the 3.0.7 templates update show up as Joomla updates the next time I open the website.
